src/backend/vulkan/src/lib.rs

```diff
@@ -647,6 +649,285 @@ impl hal::Instance<Backend> for Instance {
             .functor
             .destroy_surface(surface.raw.handle, None);
     }
+
+    fn enumerate_available_displays<'a>(&self,adapter: &'a adapter::Adapter<Backend>)->Result<Vec<hal::display::Display<'a,Backend>>,hal::device::OutOfMemory>{
+        let display_extension = ash::extensions::khr::Display::new(&self.entry,&self.raw.inner);
```

Comment:
you are right! I don't know why i thought that functions were related to the `Instance`. Maybe because `create_surface` is a `Instance` function, i thought they could be similar. I will move all that functions to `PhysicalDevice`.